src/pages/inboxPlacement/components/TestDetails.js

```diff
@@ -30,18 +32,32 @@ const TestDetails = ({ details, placementsByProvider }) => {
           </Grid>
 
         </Panel.Section>
-        <Panel.Section>
-          <Grid>
-            <Grid.Column md={8}>
-              <h3>Folder Placement</h3>
-              <FolderPlacementBarChart placements={placements}/>
-            </Grid.Column>
-          </Grid>
-        </Panel.Section>
+        <Grid>
+          <Grid.Column md={8}>
```

Comment:
Resizing the window smaller horizontally makes the charts too small. Maybe something like 
```suggestion
          <Grid.Column lg={8} md={12}>
```